netbout-web/src/test/java/com/netbout/rest/MarkdownTest.java

```diff
@@ -162,6 +162,20 @@ public void formatsBulletsToHtml() throws Exception {
     }
 
     /**
+     * Markdown can break a single line.
+     * @throws Exception If there is some problem inside
```

Comment:
@antonini Looks like that none of the calls in this test are throwing checked exceptions, please remove this and the throws declaration